### DISCUSSION TITLE
Audio offset slider's tooltip should have explanation like beatmap offset slider's tooltip

### DISCUSSION THREAD

**Discussion by DarkbitNetwork** [ppy/osu#26230] [Ideas] [answered]
I was confused about what audio offset does and there's no explanation on what it does unlike beatmap offset (which say something like "4.0 ms (make hit objects appear earlier)") so I decided to ask on discord server (see https://discord.com/channels/188630481301012481/1097318920991559880/1190301132690554941) and found the answer there.

I propose making it so audio latency slider's tooltip also explains what it does like the beatmap offset slider's tooltip. See screenshots for what I mean. 
Although unrelated, I think calling it "audio offset" is misleading because it has nothing to do with audio at all (the answer I got is that it moves the object's hit time of every beatmaps and not the audio itself)

Beatmap offset: 
![beatmap offset](https://github.com/ppy/osu/assets/43719333/a4a83b82-cea3-485e-b83d-457f7a2f653a)

Audio offset
![audio offset](https://github.com/ppy/osu/assets/43719333/e9284ed1-85a9-4160-bd43-ea7538e4310f)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/26591
